Patterns/English/English-TimeZone.yaml

```diff
@@ -16,6 +16,7 @@ AbbreviationsList: !list
     - ART
     - ADT
     - ACST
+    - AET
     - AEST
     - AEDT
     - AZT
```

Comment:
This list is not sorted. Could you also sort it, like the one for the resolutions? That really benefits checking for inconsistencies.